src/Form/IslandoraSettingsForm.php

```diff
@@ -159,6 +165,9 @@ public function buildForm(array $form, FormStateInterface $form_state) {
           $state_selector => ['checked' => TRUE],
         ],
       ],
+      '#config' => [
+        'key' => 'islandora.settings:' . self::BROKER_PASSWORD,
+      ],
```

Comment:
May additionally want the [`secret` key](https://www.drupal.org/docs/8/modules/config-override-inspector/integration-with-coi#s-secrets) here?:

```suggestion
      '#config' => [
        'key' => 'islandora.settings:' . self::BROKER_PASSWORD,
        'secret' => TRUE,
      ],
```

... though, may be irrelevant since we don't actually set the `#default_value` (or `#value`)? Not sure.

Reply:
My guess is that when configured to display the override value (which I think is sensible), the `secret` key causes the value to not be displayed in plain text.